### ISSUE TITLE
user results from WHOIS should be masked by default

### ISSUE THREAD

**Issue by bear** [prologic/eris#7]
When connecting to eris unsecurely I should not be able to see much information about other users - here is a test user `bob` connecting and then doing a `/whois bear` 

```
[S+Z] Join to #lobby is now synched (0.001 seconds)
[S+Z] bear      : bear@REDACTED.comcast.net (Mike) [Network]
[S+Z] IrcOp     : bear is an IRC operator
[S+Z] SignOn    : Fri Nov 24 21:29:01 2017    Idle : 0d 0h 26m 18s
[S+Z] Channels  : @#lobby
[S+Z] Secure    : bear is using a secure connection
[S+Z] Server    : test.bear.im (Bear's Server)
```
I would suggest that SignOn details and the IP Address be hidden and that the list of Channels be only those that are public

**Comment by prologic** [comment]
Ahh what a fantastic idea! This is definitely in line with the project
goals. Can you submit a PR? Feel free to discuss on IRC (details in the
README)


James Mills / prologic

E: prologic@shortcircuit.net.au
W: prologic.shortcircuit.net.au

**Comment by prologic** [comment]
Closed via 02427bcb3f6e651ca95a39659dfa1cacb761c574
